admin-dev/themes/new-theme/js/pages/product/components/combination-list/combinations-list-renderer.ts

```diff
@@ -278,7 +278,17 @@ export default class CombinationsListRenderer {
     const $finalPricePreview = $finalPrice.siblings(CombinationsMap.list.finalPricePreview);
     let combinationPrice = productPrice.plus(priceImpactTaxExcluded);
 
-    const ecoTax = this.productFormModel.getBigNumber('price.ecotaxTaxExcluded') ?? new BigNumber(0);
+    const combinationEcoTax = $(CombinationsMap.list.ecoTax, $row).val();
+
+    let ecoTax;
+
+    if (combinationEcoTax !== undefined) {
+      ecoTax = combinationEcoTax > 0
+        ? new BigNumber(combinationEcoTax.toString())
+        : this.productFormModel.getBigNumber('price.ecotaxTaxExcluded') ?? new BigNumber(0);
+    } else {
+      ecoTax = this.productFormModel.getBigNumber('price.ecotaxTaxExcluded') ?? new BigNumber(0);
+    }
```

Comment:
I think this way we get rid of one else statement and its still the same no?
```
if (combinationEcoTax !== undefined && combinationEcoTax > 0 ) {
  ecoTax = new BigNumber(combinationEcoTax.toString());
} else {
  ecoTax = this.productFormModel.getBigNumber('price.ecotaxTaxExcluded') ?? new BigNumber(0);
}

Reply:
Oh nice catch the code is easier to read this way!